WebUntisAPI.Client/Models/Timetable/Period.cs

```diff
@@ -109,10 +109,26 @@ public class Period
     /// </summary>
     [JsonProperty("studentGroup")]
     public string StudentGroup { get; set; } = string.Empty;
-
     /// <summary>
-    /// All elements that are associated with this period
-    /// </summary>
+    /// Returns the first part from StudentGroup
+    /// </summary>
+    public string LessonShortName
+    {
+        get
+        {
+            try
+            {
+                return StudentGroup.Split("_")[0];
+            }
+            catch
+            {
+                return string.Empty;
+            }
+        }
+        private set { }
+    }    /// <summary>
+         /// All elements that are associated with this period
+         /// </summary>
```

Comment:
The name of the lesson is available from an element of type Subject in the period. The student group is (I think) an internal value to identity it there isn't a need to process it.